yarn.lock

```diff
@@ -2564,6 +2564,19 @@
     estree-walker "^1.0.1"
     picomatch "^2.2.2"
 
+"@sentry/cli@^1.latest":
```

Comment:
My yarn lock is stale when running on this branch. Can we push the unstaled version into this PR?